assets/javascripts/kitten/components/carousel/simple-carousel.js

```diff
@@ -46,11 +68,14 @@ class SimpleCarouselBase extends Component {
         </Marger>
 
         {this.showPagination() && (
-          <Marger top="4" bottom="4" style={styles.pagination}>
+          <Marger top="4" bottom="4" style={paginationStyle}>
             {rangePage.map(numPage => {
               const pageStyle = [
                 styles.page,
-                numPage === currentPageNumber && styles.page.active,
+                paginationColor && { background: paginationColor },
+                numPage === currentPageNumber && {
```

Comment:
Pourquoi on met un `==`. On pourrait être ennuyé dans le cas suivant :
```
numPage = 3
currentPageNumber = '3'
numPage == currentPageNumber # => true
numPage === currentPageNumber # => false
```

Reply:
C'est @LanF3usT qui est à l'origine de ce composant.

Reply:
Oui ?

Reply:
Ok, in laisse tel quel 👍